ublox/src/ubx_packets/packets.rs

```diff
@@ -2358,6 +2358,97 @@ struct CfgValSet<'a> {
     cfg_data: &'a [CfgVal],
 }
 
+pub const MAX_CFG_KEYS: u16 = 64;
+
+#[ubx_packet_send]
+#[ubx(
+  class = 0x06,
+  id = 0x8b,
+  max_payload_len = 260, // 4 + sizeof(u32) * MAX_CFG_KEYS
+)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+struct CfgValGet<'a> {
+    /// Message version
+    version: u8,
+    /// The layers from which the configuration items should be retrieved
+    #[ubx(map_type = CfgReadLayer)]
+    layers: u8,
+    position: u16,
+    cfg_keys: &'a [CfgKey],
```

Comment:
Just to get some kind of association between the lifetime name and the value that determines the lifetime

```suggestion
struct CfgValGet<'k> {
    /// Message version
    version: u8,
    /// The layers from which the configuration items should be retrieved
    #[ubx(map_type = CfgReadLayer)]
    layers: u8,
    position: u16,
    cfg_keys: &'k [CfgKey],
```

Reply:
Not sure it matters that much here as we have a single lifetime specifier. But I will do it.

Reply:
Interestingly, trying to fix your suggestion made me catch this ... The lifetime needs to always be `'a` as it is hardcoded in the macro here https://github.com/ublox-rs/ublox/blob/0f9889bf9fc30ef71e13cff6610c20e2bf8beb6d/ublox_derive/src/output.rs#L449

Reply:
haha yea that's pretty ugly, it has to be part of the macro refactor at a later time imo

Reply:
I'll turn this into an issue and resolve the comment.